components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -325,8 +325,8 @@ - (void)setFloatingButtonElevation:(MDCBottomAppBarFloatingButtonElevation)float
     [_floatingButton setElevation:1 forState:UIControlStateNormal];
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, kMDCButtonAnimationDuration * NSEC_PER_MSEC),
                    dispatch_get_main_queue(), ^{
-                     [self insertSubview:_floatingButton atIndex:subViewIndex];
-                     [_floatingButton setElevation:elevation forState:UIControlStateNormal];
+                     [self insertSubview:self.floatingButton atIndex:subViewIndex];
```

Comment:
Please make this a weak reference. If the view is no longer retained, we shouldn't keep it around just to do these animations.